### PR TITLE
Feature/display tags as objects

### PR DESCRIPTION
**Pull request by danielronnkvist**
Implemented the [jsTag](http://eranhirs.github.io/jsTag/) directive for tag input.

### CODE REVIEW

File: app/assets/javascripts/controllers/FileCtrl.coffee [hunk 1]
```diff
@@ -1,18 +1,15 @@
-@bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$http', (FileHandler, $scope, $http) ->
+@bruseApp.controller 'FileCtrl', ['FileHandler', '$scope', '$rootScope', '$location', '$routeParams', (FileHandler, $scope, $rootScope, $location, $routeParams) ->
   $scope.files = []
   $scope.bruse_files = []
   $scope.new_files = []
   $scope.loading = true
 
-  # get identity information from add view
-  $scope.identity = IDENTITY_PARAMS
```
Comment (ronjagrosz) [inline app/assets/javascripts/controllers/FileCtrl.coffee:8]: In the branch feature/drive-files the IDENTITY_PARAMS changed to contain service as well. So the change to get it to match develop now is not large. $routeParams.identity_id only has to be changed to $routeParams.identity.id(?)

Comment (danielronnkvist) [inline app/assets/javascripts/controllers/FileCtrl.coffee:8]: $routeParams collects the variable from the url, you can see it [here](https://github.com/tkomstadius/bruse/pull/34/files#diff-af5144432cce9b5a21463a4f62507c60R5). The easiest merge would probably be to just skip the $routeParams everywhere and have it like it is in #32.

**Comment by danielronnkvist** [comment]
This is based upon #31, so wait until that one is merged.

File: app/assets/stylesheets/jsTag.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+/*
```
Comment (klaseskilson) [inline app/assets/stylesheets/jsTag.css:1]: shouldn't this be placed in the `vendor` folder?

Comment (klaseskilson) [inline app/assets/stylesheets/jsTag.css:1]: or maybe not... We'll probably want to change the looks of the tags.

**Comment by ronjagrosz** [comment]
:+1:

**Comment by klaseskilson** [comment]
skarmavbild 2015-04-21 kl 11 36 14

I can add tags with spaces in them, but how do I search for them if they contain spaces?

**Comment by klaseskilson** [comment]
Would it be possible to add as a tag object if I press space instead of enter?

**Comment by danielronnkvist** [comment]
True, I can look into it.

**Comment by danielronnkvist** [comment]
Tags now break on space, tab and enter.

**Comment by klaseskilson** [comment]
Looks good!

**Comment by ronjagrosz** [comment]
:rocket: ready for merge

**Pull request merged**